airflow/api_connexion/endpoints/dag_endpoint.py

```diff
@@ -86,10 +86,13 @@ def get_dags(limit, session, offset=0, only_active=True, tags=None, dag_id_patte
 
 @security.requires_access([(permissions.ACTION_CAN_EDIT, permissions.RESOURCE_DAG)])
 @provide_session
-def patch_dag(session, dag_id, update_mask=None):
+def patch_dag(session, dag_id, update_mask=None, multi=False):
```

Comment:
This `multi` flag does not looks right to me. Bulk operations need their own endpoint instead IMO.

Reply:
Maybe a `POST` to `/dags/~/patch` like `get_dag_runs_batch`, or just send a `PATCH` to `/dags/` would be a good idea. Not sure. But this is not the right solution.

Reply:
I think that this sounds like a good solution, but I would suggest that we still include a `dag_id_pattern` argument for filtering DAG IDs (which would then be consistent with the `GET /dags` endpoint)

Reply:
Agreed, an argument to filter the list endpoint sounds like a good thing to have on its own.

Reply:
Thanks for the suggestion, I agree that `patch_dags` should follow the same pattern as `get_dags`.